regtests/bin/matrix_divider.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+# --------------------------------------------------------------------------- #
+# matrix_divider simply divides the main matrix into subsets with a given     #
+# number of test (i.e. maxlist=100)                                           #
+#                                                                             #
+# Remarks:                                                                    #
+# - Once the matrix is generated, this script can be execute. The user should #
+#   define the maxlist and this script divide the matrix into matrix? and     #
+#   submit the job using sbatch.                                              #
+#                                                                             #
+#                                                      Ali Abdolali           #
+#                                                      August 2018            #
+#                                                      March 2021             #
+#                                                                             #
+#    Copyright 2013 National Weather Service (NWS),                           #
+#       National Oceanic and Atmospheric Administration.  All rights          #
+#       reserved.  WAVEWATCH III is a trademark of the NWS.                   #
+#       No unauthorized use without permission.                               #
+#                                                                             #
+# --------------------------------------------------------------------------- #
+# --------------------------------------------------------------------------- #
+# 1.  clean up and definitions                                                #
+# --------------------------------------------------------------------------- #
+rm before
```

Comment:
Perhaps also add this to the end for cleaning up?   Do we want to add these files to .gitignore?

Reply:
Sure,
Yes, we should add  them to .gitignore